docs/docs/concepts/10.actions.md

```diff
@@ -54,12 +54,11 @@ export default class extends React.Component {
 }
 ```
 
-<details>
-<summary>Example in Facebook Messenger</summary>
+### Example in Facebook Messenger
 
-![](https://botonic-doc-static.netlify.com/images/john_doe.png)
-
-</details>
+<p align="center">
+<img src="https://botonic-doc-static.netlify.app/images/john_doe.png" width="200" />
```

Comment:
The image doesn’t correspond to a Facebook messenger example, it’s from the Webchat

Reply:
Thanks, updated